packages/ember-application/lib/system/application-instance.js

```diff
@@ -79,7 +87,9 @@ let ApplicationInstance = EmberObject.extend(RegistryProxy, ContainerProxy, {
 
     var application = get(this, 'application');
 
-    set(this, 'rootElement', get(application, 'rootElement'));
+    if (!isEnabled('ember-application-visit')) {
```

Comment:
Why is this feature flagged? Application instances aren't exposed via public API if the flag is disabled, right?